pyanaconda/payload/dnfpayload.py

```diff
@@ -364,7 +364,7 @@ def _add_repo(self, ksrepo):
             repo.mirrorlist = mirrorlist
         if metalink:
             repo.metalink = metalink
-        repo.sslverify = not (ksrepo.noverifyssl or flags.noverifyssl)
+        repo.sslverify = not (ksrepo.noverifyssl or not self.is_ssl_verification_enabled)
```

Comment:
I don't think this is correct. This will check `method.noverifyssl` for additional repositories now.

Reply:
True. Removed.